packages/rs-sdk/src/platform/transition/txid.rs

```diff
@@ -0,0 +1,22 @@
+use crate::Sdk;
+
+/// State transition identifier
+pub struct TxId([u8; 32]);
```

Comment:
I think it would be best to just say TxId is a Bytes32, and get all the benefit from that already defined struct.

Reply:
It also should be in DPP and not the SDK